networking/rosa-shared-vpc-config.adoc

```diff
@@ -0,0 +1,29 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-shared-vpc-config"]
+= Configuring a shared virtual private cloud for ROSA clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can create {product-title} clusters into shared, centrally-managed AWS virtual private clouds (VPCs). This process requires two separate AWS accounts belonging to the same AWS organization. One account functions as the VPC owner while the other account creates the cluster in this VPC.
```

Comment:
Is "into" the right preposition with "create"? Usually I would expect "in" or "on" or something similar.